macos/FluentUI/Link.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import AppKit
+
+/// A lightweight hyperlink control
+@objc(MSFLink)
+open class Link : NSButton {
+	
+	/// Initialies a link with a content string that is displayed, and an underlying
+	/// - Parameters:
+	///   - content: The visible text of the link that the user sees.
```

Comment:
Interesting thing to address - what if someone wants to change the title later? What do they do? Maybe saying in the comments that it is not currently supported is best path?

Reply:
I guess we could override didSet of title and route it through attributedTitle with our link attributes? If we wanted to enforce that titles on this button have the link style.

Reply:
Yep! Ended up overriding the title property. I could make setting the alternateTitle/attributedTitle a `preconditionFailure`, but I'll go with the earlier suggestion of "give clients a nice default, and let them do what they like afterwards"